src/Library/demos/Link Button/code.rs

```diff
@@ -0,0 +1,15 @@
+use crate::workbench;
+use gtk::prelude::*;
+
+pub fn main() {
+    let linkbutton: gtk::LinkButton = workbench::builder().object("linkbutton").unwrap();
+    linkbutton.connect_clicked(move |_| {
```

Comment:
The javascript version uses https://gtk-rs.org/gtk4-rs/stable/latest/docs/gtk4/struct.Button.html#method.connect_notify with the parameter "visited" here.

Reply:
I tried with connect_notify first but could not resolve errors, so used an alternative.
It says "closure takes 2 arguments but is given 1" even though i give it 2 arguments.
Ill try using connect_notify and resolve this

Reply:
Hello, I've added the method you specified.

Please review :)